packages/react/src/codemods/codemods-helpers.tsx

```diff
@@ -21,11 +23,14 @@ export const getImportDeclarationCollection = ({
   j: JSCodeshift;
   collection: Collection<any>;
   importPath: string;
-}) =>
-  collection
+}): Collection<ImportDeclaration> => {
```

Comment:
Love explicit typings over implicit one. TS compiler will be fast if we specify return types.

Reply:
we could add an eslint rule for that too.

Reply:
Maybe later. Should we add card for this? Because it might require changes too.

Reply:
added https://github.com/atlassian-labs/compiled/issues/422